app/models/administrative-unit.js

```diff
@@ -76,15 +76,25 @@ export default class AdministrativeUnitModel extends OrganizationModel {
         IGSCodeList,
         REQUIRED_MESSAGE
       ),
-      wasFoundedByOrganizations: validateRequiredWhenClassificationId(
-        [
-          ...AgbCodeList,
-          ...ApbCodeList,
-          ...OcmwAssociationCodeList,
-          ...PevaMunicipalityCodeList,
-          ...PevaProvinceCodeList,
-        ],
-        REQUIRED_MESSAGE
+      wasFoundedByOrganizations: Joi.when(
+        Joi.ref('$relaxMandatoryFoundingOrganization'),
```

Comment:
**Nitpick:** What about adding a comment to explain were `$relaxMandatoryFoundingOrganization` is coming from?